src/pages/Detail/component/DetailIntro.jsx

```diff
@@ -0,0 +1,232 @@
+import React, { useState, useEffect, useCallback } from 'react';
+import styled from 'styled-components';
+import Modal from 'react-modal/lib/components/Modal';
+import Creator from './Creator';
+import Review from './Review';
+import ReviewModal from './ReviewModal';
+import { AiOutlineClose } from 'react-icons/ai';
+
+function DetailIntro({
+  image,
+  classTitle,
+  classIntroduce,
+  creatorName,
+  profile_image,
+}) {
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const [reviews, setReviews] = useState([]);
+
+  const showModal = () => {
+    setIsModalOpen(true);
+  };
+
+  const closeModal = useCallback(() => {
+    setIsModalOpen(false);
+  }, []);
+
+  useEffect(() => {
+    fetch(`http://localhost:3000/data/data_review.json`)
```

Comment:
```suggestion
    fetch(`/data/data_review.json`)
```
목데이터로 구현시 localhost:3000은 생략 가능합니다!